js_tests/tests.html

```diff
@@ -78,12 +78,17 @@
     <script src='../django/contrib/admin/static/admin/js/prepopulate.js' data-cover></script>
     <script src='../django/contrib/admin/static/admin/js/urlify.js' data-cover></script>
 
-    <div id="id_point_map">
+    <div id="id_point_map" style="display:none;">
         <textarea id="id_point" name="point"
                   class="vSerializedField required" style="display:none;"
-                  rows="10" cols="150">POINT (7.8177 47.397)</textarea>
+                  rows="10" cols="150">{&quot;type&quot;: &quot;Point&quot;, &quot;coordinates&quot;: [7.8177, 47.397]}</textarea>
     </div>
-    <script src='http://openlayers.org/api/2.13.1/OpenLayers.js'></script>
+    <div id="id_multipolygon_map" style="display:none;">
+        <textarea id="id_multipolygon" name="multipolygon"
+                  class="vSerializedField required" style="display:none;"
+                  rows="10" cols="150"></textarea>
+    </div>
+    <script src='//cdnjs.cloudflare.com/ajax/libs/ol3/3.15.1/ol.js'></script>
```

Comment:
Did you try running the tests with `npm test`? Part of the issue seems to be using `//` here. After switching to `https://` the tests get further along. I can debug further if you'd like.